### PR TITLE
LPD-21412 Add warning message when updatting a category used in a URL

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPD-21412)

## Steps

*   Go to Categorization > Categories
*   Create a Vocabulary
*   Create a Category and delete it

deleteCat

*   To test this part we need [this story](https://liferay.atlassian.net/browse/LPD-6814) (Including the Category in the Blog Post URL) to be merged

InfoRelatedCat

### CODE REVIEW

**Review by jkappler — CHANGES_REQUESTED** [review]

**Comment by DiegoHu97** [comment]
Changes made, thanks for the review :love\_you\_gesture:

**Comment by DiegoHu97** [comment]
ci:test:sf

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
@DiegoHu97

File: modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/category/details.jsp [hunk 1]
```diff
@@ -56,6 +56,13 @@ renderResponse.setTitle(title);
 
 		<aui:model-context bean="<%= category %>" model="<%= AssetCategory.class %>" />
 
+		<c:if test="<%= (assetCategoriesDisplayContext.getAssetEntryAssetCategoryRelsCountByClassNameId(categoryId, PortalUtil.getClassNameId(FriendlyURLEntry.class)) > 0) && (category != null) %>">
```
Comment (jkappler) [inline modules/apps/asset/asset-categories-admin-web/src/main/resources/META-INF/resources/category/details.jsp:59]: @DiegoHu97, I forgot that we should hide behind FF 😞

**Review by jkappler — CHANGES_REQUESTED** [review]

**Comment by DiegoHu97** [comment]
ci:test:sf

**Review by jkappler — APPROVED** [review]

**Comment by jkappler** [comment]
ci:forward

**Comment by jkappler** [comment]
ci:report:2561460946

**Comment by jkappler** [comment]
ci:forward:force

**Comment by DiegoHu97** [comment]
ci:forward:force

**Pull request closed without merging**